pkg/objectcache/networkneighborhoodcache/networkneighborhoodcache.go

```diff
@@ -78,6 +81,215 @@ func (nn *NetworkNeighborhoodCacheImpl) GetNetworkNeighborhood(containerID strin
 	return nil
 }
 
+func (nn *NetworkNeighborhoodCacheImpl) handleUserManagedNN(netNeighborhood *v1beta1.NetworkNeighborhood) {
```

Comment:
I assume this one is copy/paste from AP (if not keep it similar so we can verify we have the same handling between AP and NN)